.github/workflows/docs.yml

```
@@ -3,13 +3,13 @@ name: Deploy Documentation
 on:
   push:
     branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)